web-app/src/index.css

```
@@ -1,3 +1,228 @@
+  }
+
+  .react-tags.is-active {
+    border-color: var(--color-primary);
+  }
+
+  .react-tags__label {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    overflow: hidden;
+    clip: rect(0 0 0 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)